content/docs/custom-commands/commands.md

```diff
@@ -242,6 +249,110 @@ The modal trigger is used to trigger custom commands via submitting a modal.
 
 The trigger is matched using [RegEx](/docs/reference/regex).
 
+##### Crontab
+
+This trigger will run the command periodically using [Cron Syntax](https://en.wikipedia.org/wiki/Cron) to schedule runs.
+In contrast to interval triggers, which run a command with a fixed delay but unknown time, cron triggers can be made to
+execute periodically at fixed times, dates, and/or intervals. For instance, you could schedule execution for 23:45 every
+Saturday.
+
+When using a time-based trigger, the custom command does not receive any user or member context. Thus, `{{ .User.ID }}`
+and similar templates will result in no value and member-dependent functions such as `addRoleID` will fail.
+
+![Overview of crontab configuration options.](crontab_trigger_options.png?width=60vw)
+
+<center>
+
+**1** Cron Expression **2** Channel **3** Excluding hours/weekdays
+
+</center>
+
+Cron Expression (**1**) defines the expression used to schedule the cron job. It uses the standard expression format
+(see below). It does not support predefined schedules such as `@hourly`. The cron scheduler uses UTC always.
+
+A cron expression represents a set of times, using 5 space-separated fields.
+
+Field name   | Mandatory? | Allowed values  | Allowed special characters
+----------   | ---------- | --------------  | --------------------------
+Minutes      | Yes        | 0-59            | * / , -
+Hours        | Yes        | 0-23            | * / , -
+Day of month | Yes        | 1-31            | * / , - ?
+Month        | Yes        | 1-12 or JAN-DEC | * / , -
+Day of week  | Yes        | 0-6 or SUN-SAT  | * / , - ?
+
+To read more about the supported format of cron expressions, visit [Robfig's Cron package documentation - Expression
+Format](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-CRON_Expression_Format).
+
+{{< callout context="tip" title="Tip: Debugging Cron Expressions" icon="outline/rocket" >}}
+
+To help build and debug cron expressions, we reccomend using [Cronitor's Crontab Guru](https://crontab.guru/) or a
+similar site. Note that predefined schedules such as `@hourly`, and the use of `7` in the DOW field may parse correctly
```

Comment:
You should introduce this abbreviation in the table above; perhaps also for DOM.